masterfabric-mobile/swift-camp
# CL - ActionSheet Viper Implementation

## 📋 PR Description

This pull request implements the ActionSheetView within the VIPER architecture, ensuring a clean and modular design. The ActionSheetView component demonstrates dynamic action sheet handling with user interactions, allowing users to select various options seamlessly. The changes include modular components, clean code refactoring, and comprehensive testing for a smooth user experience.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Open the project in Xcode and ensure all dependencies are resolved.
2. Navigate to the module or screen where the ActionSheetView is implemented.
3. Trigger the ActionSheet by interacting with the designated button.
4. Look all ActionSheet options like; Option 1, Option 2, Delete and Cancel
5. Verify that the ActionSheet dismisses correctly after an option is selected.
6. Run the application and ensure there are no layout issues or unexpected crashes.

---

## 🔗 Related Links

- Documentation:  https://developer.apple.com/documentation/swiftui/actionsheet

---

### 📷 Screenshots (Optional)

<img width="763" alt="ActionSheet" src="https://github.com/user-attachments/assets/4192a92b-88ec-432b-a5e0-844f0b739346" />
